modules/fusb302b.py

```diff
@@ -462,9 +462,9 @@ def get_interrupts(self):
         Interruptb = self.i2c.readfrom_mem(
             self.ADDRESS, self.good_crc_sent_int.register, 1
         )[0]
-        Interrupt = self.i2c.readfrom_mem(
-            self.ADDRESS, self.bc_level_int.register, 1
-        )[0]
+        Interrupt = self.i2c.readfrom_mem(self.ADDRESS, self.bc_level_int.register, 1)[
+            0
+        ]
```

Comment:
I think this is the more generally accepted way of splitting this line
```suggestion
        Interrupt = self.i2c.readfrom_mem(
            self.ADDRESS, self.bc_level_int.register, 1)[0]
```

Reply:
Agreed, i tried this (and the ones above are more readable than the autogenerated one here imo) but ruff keeps reformatting it back to its preferred option - are we happy enough to leave this as is and just merge accepting it looks a little odd?

Reply:
Funny. Sure feel free to merge